src/raw/request.rs

```diff
@@ -35,7 +35,7 @@ pub struct Payload<Track: StrType, Artist: StrType = Track, Release: StrType = T
 /// Type of the [`Payload::track_metadata`] field.
 ///
 /// If [`release_name`](Self::release_name) will always be [None] and the type for `Release` cannot be inferred, set it to the [Empty] type.
-#[derive(Debug, serde::Serialize)]
```

Comment:
This is my only nitpick: I would rather this change to the canonical path for the `serde` Trait is in a separate PR.

Other than that, I am happy to approve.

Reply:
Imo we should just merge, the change is too small for a separate PR

Reply:
Fair enough. Let's go ahead with a merge, then :)